### DISCUSSION TITLE
3.6.2

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#57] [Announcements]
Blender 3.6.2, commit [e53e55951e7a](https://projects.blender.org/blender/blender/commit/e53e55951e7a).
Build environment:

    Visual Studio 2019, version 16.11.29
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.6.2'>3.6.2</a>.</em>

**Comment by qqato** [comment]
下載不了
